registration/templates/registration/activation_complete_admin_pending.html

```diff
@@ -0,0 +1,22 @@
+{% extends "registration/registration_base.html" %}
```

Comment:
what do you mean? If you use the example app to go through the registration flow just take some screenshots of what these pages look like, it's an easy way to catch small bugs on the templates in my opinion